src/components/IntegrationsView/IntegrationForm/IntegrationForm.tsx

```diff
@@ -23,19 +19,13 @@ import {
   CustomLabel,
 } from "../CustomFieldsAndWidgets";
 import { toPascalCase } from "../../../utils/toPascalCase";
-import {
-  useAppDispatch,
-  useAppSelector,
-  useEventsBusForIDE,
-  useSendChatRequest,
-} from "../../../hooks";
-import {
-  setIsConfigFlag,
-  setToolUse,
-} from "../../../features/Chat/Thread/actions";
+import { type SmartLink } from "../../../services/refact";
```

Comment:
This import could live near Integration, IntegrationField and IntegrationPrimitive types from `../../../services/refact`